tools/checkimports.py

```diff
@@ -41,5 +41,10 @@ def is_valid(file_name):
     return valid
 
 if __name__ == '__main__':
-  files = glob.iglob(sys.argv[1] if len(sys.argv) > 1 else 'src/*.cc')
+  if len(sys.argv) > 1:
+    files = []
+    for pattern in sys.argv[1:]:
+      files = itertools.chain(files, glob.iglob(pattern))
+  else:
+    files = glob.iglob('src/*.cc')
   sys.exit(0 if all(map(is_valid, files)) else 1)
```

Comment:
I think what may be happening in the CI is that `all()` is short circuiting and `is_valid` doesn't run on all of the files returned by the glob(s). 
```suggestion
  sys.exit(0 if all(list(map(is_valid, files))) else 1)
```
may fix that, or there may be a more Pythonic way of ensuring we run the function against every item returned by the iterator(s) to get all the warnings printed. cc @nodejs/python